README.md

```diff
@@ -1,2 +1,54 @@
+
+**Effective Date:** Fri Mar 15 2024
+
+This Privacy Policy applies to the Chrome extension developed for internal use by Plume ("we", "us", or "our"). The extension is designed exclusively for all personnel including employees, consultants, contractors of both Plume and Plume Health Medical Group ("you" or "your", collectively "Plume personnel") and is not available to the general public. This policy describes how we collect, use, and protect your information in connection with the extension and is intended to comply with the Health Insurance Portability and Accountability Act (HIPAA) and Google Chrome Store's requirements.
+
+### 1.1 Information We Collect
+
+#### 1.1.a Authentication Data
+To verify your identity and maintain security, we collect authentication data, including but not limited to usernames, password hashes, and security tokens. This information is essential for granting access and ensuring that the extension and its functionalities are exclusively used by authorized Plume personnel.
+
+#### 1.1.b Access Logs
+In accordance with HIPAA regulations, we log access to the extension to monitor usage and safeguard against unauthorized access. These logs include date, time, user identification, and the nature of the accessed information.
+
+#### 1.1.c Browser Paths
+The extension requires access to the hostnames and paths of the websites you visit while using the browser. This information enables us to determine when to show the extension based on the third-party tools you are using.
+
+### 1.2 Use of Information
+We use the collected information to:
+
+- Verify user identity and authorize access.
+- Log access for security audits and compliance with HIPAA and other regulatory requirements.
+- Enhance the extension's functionality by determining its availability on certain hostnames.
+
+### 1.3 Information Sharing and Disclosure
+We do not share or disclose any collected information with third parties, except as required by law or to comply with legal processes. Access to the information is strictly limited to authorized Plume personnel who need the information to perform their job functions related to the maintenance and operation of the extension.
+
+### 1.4 Data Protection
+We are committed to protecting your information through a combination of physical, administrative, and technical safeguards. These measures are designed to ensure the confidentiality, integrity, and availability of your and your patients' information in accordance with HIPAA standards.
+
+### 1.5 Changes to This Privacy Policy
+We may update this Privacy Policy from time to time to reflect changes in our practices or legal requirements. We will notify you of any material changes by posting the updated policy on our internal communication channels.
+
+### 1.6 Contact Us
+If you have any questions about this Privacy Policy or our information practices, please contact us at it@getplume.co
```

Comment:
If you have any questions about this Privacy Policy or our information practices, please contact us at it-admin@getplume.co